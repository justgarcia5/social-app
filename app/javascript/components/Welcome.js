import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route
} from 'react-router-dom';
import Conversations from './Conversations';

const Welcome = () => {
  return (
    <>
      <h1>Display Active Users Account Details</h1>
      <Conversations />
    </>
  )
};

export default Welcome;