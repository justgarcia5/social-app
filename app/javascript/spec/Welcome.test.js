import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
// import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { mount, shallow, render } from 'enzyme';
Enzyme.configure({ adapter: new Adapter() });

import Welcome from '../components/Welcome';

describe('Welcome', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Welcome debug />);

    expect(component).toMatchSnapshot();
  });

  it('should render correctly with no props', () => {
    const component = shallow(<Welcome/>);

    expect(component).toMatchSnapshot();
  });

  it('should render banner text correctly with given strings', () => {
    const strings = ['one', 'two'];
    const component = shallow(<Welcome list={strings} />);
    expect(component).toMatchSnapshot();
  });
});