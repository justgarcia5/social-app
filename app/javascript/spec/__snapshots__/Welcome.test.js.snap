// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should render banner text correctly with given strings 1`] = `
<h1>
  Display Active Users Account Details
</h1>
`;

exports[`Welcome should render correctly in "debug" mode 1`] = `
<h1>
  Display Active Users Account Details
</h1>
`;

exports[`Welcome should render correctly with no props 1`] = `
<h1>
  Display Active Users Account Details
</h1>
`;
